appveyor.yml

```diff
@@ -24,6 +31,10 @@ build_script:
 
 after_build:
   - ps: $PebbleProcess = Start-Process pebble -PassThru
+  - mkdir deploy
+  # Two following lines are copying built executable to the proper artifacts folder
+  - copy %USERPROFILE%\go\bin\pebble.exe deploy\pebble_windows-amd64.exe
+  - copy %USERPROFILE%\go\bin\pebble-challtestsrv.exe deploy\challtestsrv_windows-amd64.exe
```

Comment:
ditto here: re `pebble-challtestsrv` (and I suspect the `artifacts` section earlier in this file will need a tweak too)

Reply:
Done here also.